From 1545f8bfdc5defd20b069310255e0f6148c4bf21 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Tue, 29 Jan 2019 15:34:50 +0100
Subject: [PATCH 0922/1051] Documentation/bindings: marvell: add CN913x board
 descriptions

Change-Id: I84cdf79a27ee9095c7d4e654560c4603114ff1ea
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/2964
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Kostya Porotchkin <Kostya.Porotchkin@cavium.com>
Reviewed-by: Nadav Haklai <Nadav.Haklai@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 .../bindings/arm/marvell/cn913x.txt           | 25 +++++++++++++++++++
 1 file changed, 25 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/arm/marvell/cn913x.txt

diff --git a/Documentation/devicetree/bindings/arm/marvell/cn913x.txt b/Documentation/devicetree/bindings/arm/marvell/cn913x.txt
new file mode 100644
index 000000000000..5fdaed39db36
--- /dev/null
+++ b/Documentation/devicetree/bindings/arm/marvell/cn913x.txt
@@ -0,0 +1,25 @@
+Marvell OcteonTX2 CN913X Platforms Device Tree Bindings
+-------------------------------------------------------
+
+Boards using a SoC of the Marvell CN913X families must carry the following
+root node property:
+
+ - compatible, with one of the following values:
+
+SoC:
+   - "marvell,cn9130", "marvell,armada-ap807-quad", "marvell,armada-ap807"
+      when the SoC being used is the CN9130
+
+Boards:
+   - "marvell,cn9130-db-A"
+      when the board being used is the OcteonTX2 CN9130 Development board with
+      Setup A.
+
+   - "marvell,cn9131-db-A"
+      when the board being used is the OcteonTX2 CN9131 Development board with
+      Setup A.
+
+Example:
+
+compatible = "marvell,cn9131-db-A", "marvell,cn9131-db",
+	     "marvell,armada-ap807-quad", "marvell,armada-ap807";
-- 
2.17.1

